examples/hybrid/driver.jl

```diff
@@ -3,18 +3,28 @@ if !(@isdefined parsed_args)
     (s, parsed_args) = parse_commandline()
 end
 
-# Distributed log must be configured before we start logging:
-# https://github.com/CliMA/ClimaAtmos.jl/issues/838
-include("config_log.jl")
-const comms_ctx = get_comms_ctx()
 if haskey(ENV, "CLIMACORE_DISTRIBUTED")
-    const pid, nprocs = ClimaComms.init(comms_ctx)
-    @info "Setting up distributed run on $nprocs \
-        processor$(nprocs == 1 ? "" : "s")"
+    if ENV["CLIMACORE_DISTRIBUTED"] == "MPI"
+        using ClimaCommsMPI
+        const comms_ctx = ClimaCommsMPI.MPICommsContext()
+    else
+        error("ENV[\"CLIMACORE_DISTRIBUTED\"] only supports the \"MPI\" option")
+    end
+else
+    const comms_ctx = ClimaComms.SingletonCommsContext()
 end
+ClimaComms.init(comms_ctx)
+
 import Logging
-atexit() do
-    Logging.global_logger(Logging.global_logger(get_logger(comms_ctx)))
+if ClimaComms.iamroot(comms_ctx)
+    Logging.ConsoleLogger(stderr, Logging.Info)
+else
+    Logging.NullLogger()
+end
```

Comment:
What is this doing? It doesn't seem like the logger is being assigned to any variables or used anywhere

Reply:
Can we pass in communicator `context` as an input argument to the hybrid driver?

Reply:
oh right